package.json

```
@@ -53,37 +53,36 @@
     "preversion": "git pull && yarn check --integrity && yarn security-check && yarn lint && yarn test",
     "postversion": "git push --tags origin HEAD",
     "prepublishOnly": "npm run lint && npm test && npm run security-check && npm run build",
-    "security-check": "nsp check",
+    "security-check": "yarn audit",
     "upgrade-dependencies": "yarn upgrade-interactive --latest --exact",
     "commitmsg": "commitlint -e $GIT_PARAMS",
     "semantic-release": "semantic-release"
   },
   "dependencies": {},
   "devDependencies": {
-    "@commitlint/cli": "7.2.1",
-    "@commitlint/config-conventional": "7.1.2",
-    "@types/aws-lambda": "8.10.15",
-    "@types/jest": "23.3.10",
-    "@types/node": "9.6.40",
-    "aws-sdk-mock": "4.3.0",
-    "coveralls": "3.0.2",
-    "husky": "0.14.3",
-    "jest": "23.6.0",
-    "lint-staged": "8.1.0",
-    "nsp": "3.2.1",
-    "nyc": "13.0.1",
-    "prettier": "1.15.3",
-    "rollup": "0.67.4",
-    "rollup-plugin-hashbang": "1.0.1",
-    "rollup-plugin-node-resolve": "3.4.0",
-    "semantic-release": "15.12.4",
-    "ts-jest": "23.10.5",
-    "tslint": "5.11.0",
-    "tslint-functional-preset": "1.5.0",
-    "typescript": "3.0.3"
+    "@commitlint/cli": "17.0.1",
+    "@commitlint/config-conventional": "17.0.0",
+    "@rollup/plugin-node-resolve": "13.3.0",
+    "@types/aws-lambda": "8.10.97",
+    "@types/jest": "27.5.1",
+    "@types/node": "17.0.35",
+    "aws-sdk-mock": "5.7.0",
+    "coveralls": "3.1.1",
+    "husky": "8.0.1",
+    "jest": "28.1.0",
+    "lint-staged": "12.4.2",
+    "nyc": "15.1.0",
+    "prettier": "2.6.2",
+    "rollup": "2.74.1",
+    "rollup-plugin-hashbang": "3.0.0",
+    "semantic-release": "19.0.2",
+    "ts-jest": "28.0.3",
+    "tslint": "6.1.3",
+    "tslint-functional-preset": "2.0.0",
+    "typescript": "4.7.2"
   },
   "peerDependencies": {
-    "aws-sdk": "^2.197.0"
+    "aws-sdk": "^2.800.0"
```

This version is without security issues
```suggestion
    "aws-sdk": "^2.814.0"
```